### PR TITLE
Bring ValueTriple into alignment with upcoming rev06

### PR DESCRIPTION
**Pull request by yogeshbdeshpande**
This reverts commit be7ec48294794682efe017eeae95ea00f6aeac56.

### CODE REVIEW

File: comid/example_psa_refval_test.go [hunk 1]
```diff
@@ -20,7 +20,6 @@ func Example_psa_refval() {
 		panic(err)
 	}
 
-	// output:
```
Comment (thomas-fossati) [inline comid/example_psa_refval_test.go:23]: if you remove the `output` keyword, I am not sure these tests are still active.

File: comid/measurement.go
Comment (thomas-fossati) [inline comid/measurement.go:1]: Not sure why these changes are needed.  I suggest to revert this file to the original.

File: comid/test_vars.go [hunk 2]
```diff
@@ -349,7 +349,7 @@ var (
 	}
 }
 `
-	CCARefValJSONTemplate = `{
+	CCARefValJSONTemplate = ` {
```
Comment (thomas-fossati) [inline comid/test_vars.go:352]: spurious space

File: comid/test_vars.go [hunk 3]
```diff
@@ -516,6 +516,7 @@ var (
 			]
 		}
 	]
+ }
```
Comment (thomas-fossati) [inline comid/test_vars.go:519]: indent should be +1

File: comid/test_vars.go [hunk 4]
```diff
@@ -516,6 +516,7 @@ var (
...
 }
-}`
+`
```
Comment (thomas-fossati) [inline comid/test_vars.go:521]: this change looks spurious

**Review by thomas-fossati — COMMENTED** [review]
Thanks! LGTM. I have left a few editorial comments inline.

File: comid/example_psa_refval_test.go (hunk 1 above)
Comment (yogeshbdeshpande) [reply comid/example_psa_refval_test.go:23]: Ah, ok, good catch,I will fix this!

File: comid/test_vars.go (hunk 4 above)
Comment (yogeshbdeshpande) [reply comid/test_vars.go:521]: This is how it is in the latest commit, so I just retained it!

File: comid/measurement.go
Comment (yogeshbdeshpande) [reply comid/measurement.go:1]: Sorry not clear what you meant? The changes in measurement.go line 776 onwards  was present earlier, for multiplicity which is now restored as part of reverting the commit?

**Review by yogeshbdeshpande — COMMENTED** [review]
See my comments, I have incorporated your edits and also replied to your question!

**Review by thomas-fossati — APPROVED** [review]
LGTM
